src/components/content/catalog/services/update/UpdateService.tsx

```diff
@@ -165,14 +170,18 @@ function UpdateService({
         return false;
     };
 
+    const onChange = (e: CheckboxChangeEvent) => {
+        setIsRemoveServiceTemplateUntilApproved(e.target.checked);
+    };
+
     return (
         <div className={catalogStyles.updateUnregisterBtnClass}>
             <Button
                 type='primary'
                 icon={<EditOutlined />}
                 onClick={showModal}
                 className={catalogStyles.catalogManageBtnClass}
-                disabled={isViewDisabled}
+                disabled={isViewDisabled || activeServiceDetail.serviceTemplateRegistrationState === 'in-review'}
             >
```

Comment:
The update request not allowed only when  `isUpdatePending` of the service template is true.